com/win32comext/mapi/src/mapi_headers/README.md

```diff
@@ -1,61 +1,67 @@
-These MAPI headers were downloaded from https://www.microsoft.com/en-us/download/details.aspx?id=12905
+These MAPI headers were downloaded from <https://www.microsoft.com/en-us/download/details.aspx?id=12905>
 and enable us to build the MAPI extension in Python 2.7 and up.
 
 The terms for these headers are below:
 
-MICROSOFT SOFTWARE LICENSE TERMS
-MICROSOFT OUTLOOK 2010 MAPI HEADER FILES
+---

```

Comment:
I like the rest of the PR, but I think copied license text really should remain true to the source.